pkg/schema/schema.go

```diff
@@ -152,6 +158,25 @@ func getType(node yamlmeta.ValueHoldingNode) (yamlmeta.Type, error) {
 	return typeOfValue, nil
 }
 
+func getExplicitDefaultValue(node yamlmeta.ValueHoldingNode) (interface{}, error) {
+	anns, err := collectAnnotations(node)
+	if err != nil {
+		return nil, NewSchemaError("Invalid schema", err)
+	}
+	annsCopy := append([]Annotation{}, anns...)
+	if len(annsCopy) == 0 {
+		return nil, nil
+	}
```

Comment:
this copies a slice into a new slice, but that doesnt actually copy deep -- dont need this code.